lib/http/request.js

```diff
@@ -27,6 +27,9 @@ const __defaultSettings__ = {
 
 let __globalSettings__ = null;
 
+let __httpKeepAliveAgent__ = null;
+let __httpsKeepAliveAgent__ = null;
```

Comment:
It would be more appropriate to use static variables within the class.

```javascript
class Abc {
  static __httpKeepAliveAgent__ = null
}
class Abc {
  // for private static variable
  static #httpKeepAliveAgent= 0
  
   // for public static variable
  static httpKeepAliveAgent = null;
  
  static getHttpKeepAliveAgent() {
    //In case of the public method one can use `Abc.httpKeepAliveAgent`
    return Abc.#httpKeepAliveAgent
  }
}
```

Reply:
it works in v14.20.0 version
![image](https://github.com/nightwatchjs/nightwatch/assets/8705386/2ade6230-e9ef-4367-875c-ce6327a53d14)

Reply:
I don't think it is necessary at the moment. This feature might pass, but there might be something else which might fail later on. If we set `ecmaVersion` to `2022` we might not catch those. We can refactor when we move to higher versions of node.

Reply:
> I don't think it is necessary at the moment. This feature might pass, but there might be something else which might fail later on. If we set ecmaVersion to 2022 we might not catch those. We can refactor when we move to higher versions of node.

Refactoring now will help us in faster typescript migration, and moreover, this doesn't break in 14.20. This feature is available from v12.x onwards
Ref: https://github.com/tc39/proposal-class-fields